lib/speech/speech_synthesizer.ts

```diff
@@ -118,16 +118,19 @@ export default class SpeechSynthesizer extends events.EventEmitter {
         return 150;
     }
 
-    private _ensureOutputStream(result : { sampleRate : number, numChannels : number }) {
+    private _closeOutputStream() {
+        if (this._outputStream)
+            this._outputStream.end();
+        this._outputStream = null;
         if (this._closeTimeout)
             clearTimeout(this._closeTimeout);
-        this._closeTimeout = setTimeout(() => {
-            if (this._outputStream)
-                this._outputStream.end();
-            this._outputStream = null;
-            this._closeTimeout = null;
-        }, 60000);
+        this._closeTimeout = null;
+    }
 
+    private _ensureOutputStream(result : { sampleRate : number, numChannels : number }) {
+        if (this._closeTimeout)
+            clearTimeout(this._closeTimeout);
+        this._closeTimeout = setTimeout(this._closeOutputStream, 60000);
```

Comment:
If we close immediately after we're done speaking we don't need the timeout logic right?

Reply:
Right, removed all the references